jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/FileBufferedResponseHandler.java

```diff
@@ -229,5 +230,23 @@ protected void onFailure(Throwable cause)
             };
             icb.iterate();
         }
+
+        private ByteBuffer readByteBuffer(Path path, long pos, long len) throws IOException
+        {
+            ByteBuffer buffer = BufferUtil.toMappedBuffer(path, pos, len);
+            if (buffer != null)
+                return buffer;
+
+            // if we reached here, then file mapped byte buffers is not supported.
+            // we fall back to using traditional I/O instead.
+            try (SeekableByteChannel channel = Files.newByteChannel(path))
+            {
+                channel.position(pos);
+                buffer = ByteBuffer.allocate((int)len);
```

Comment:
Please use direct byte buffers by default.

Reply:
Addressed this with the new internal `SendFileCallback` class